### PR TITLE
refactor onyx workflow

### PR DESCRIPTION
**Pull request by idigs**
set version for octokit request github action module

https://github.com/octokit/request-action

### CODE REVIEW

**Review by mbeidler3 — APPROVED** [review]

**Review by idigs — COMMENTED** [review]
This looks ready

**Comment by mbeidler3** [comment]
Test this please

**Comment by idigs** [comment]
test please

File: .github/workflows/onyx.yml [hunk 1]
```diff
@@ -28,12 +28,12 @@ jobs:
 
 
   job_setup_workdir:
-    runs-on: self-hosted
+    runs-on: [ self-hosted ]
     needs: [ job_verify_actor ]
     steps:
       - name: GitHub API Request
         id: api_request
-        uses: octokit/request-action@v2.1.7
```
Comment (idigs) [inline .github/workflows/onyx.yml:36]: This was the critical change that allows the workflow dir setup

**Review by idigs — COMMENTED** [review]
Change the version for octokit/request-action

**Review by quantumsteve — APPROVED** [review]

**Pull request merged**